setup.py

```diff
@@ -46,23 +45,25 @@ def join_args(args):
     return ' '.join(map(shlex.quote, args))
 
 
-class CMakeExtension(Extension):
-    def __init__(self, name):
-        Extension.__init__(self, name, sources=[])
-
-
 class OTIO_build_ext(setuptools.command.build_ext.build_ext):
     """
     def initialize_options(self):
         super(setuptools.command.build_ext.build_ext, self).initialize_options()
     """
 
+    built = False
+
     def run(self):
+        self.announce('running OTIO build_ext', level=2)
+        super().run()
```

Comment:
This is rather important. `build_ext.run` takes care of a couple of things, one of which is to copy the built files into the source tree (in `src/py-opentimelineio/opentimelineio`).